cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2110,6 +2110,50 @@ action.cyences_notable_event_action.attacker_drilldown = `cs_o365` sourcetype="o
 action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Office 365
```

Comment:
Add new parameters called:

- action.cyences_notable_event_action.deprecated = true" (default=false)
- action.cyences_notable_event_action.descrecated_from_version =  4.0.0 (default = empty)
- action.cyences_notable_event_action.deprecated_replacement = <full name of alert> (here "O365 - Login Failure From New Location Due To Multi Factor Authentication")

Follow for other alerts as we have a plan to remove them in the future.